src/main/java/hudson/plugins/git/extensions/impl/CleanCheckout.java

```diff
@@ -17,8 +17,19 @@
  * @author Kohsuke Kawaguchi
  */
 public class CleanCheckout extends GitSCMExtension {
+
+    private boolean cleanSubmodule;
     @DataBoundConstructor
+    public CleanCheckout(final boolean cleanSubmodule) {
```

Comment:
See the [comment](https://github.com/jenkinsci/git-plugin/pull/449/files#r242635862) on `CleanBeforeCheckout`.